src/index.html

```diff
@@ -6,8 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/rating.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a href="#" class="card__link">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="card__img"
+        >
+      </a>
+
+      <h2 class="card__titel">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
+      <span class="card__code">Product code: 195434</span>
+
+
+
+      <div class="card__rating rating">
```

Comment:
```suggestion
      <span class="card__code">Product code: 195434</span>

      <div class="card__rating rating">
```
1 blank line is enough if you want to add indents between lines